packages/st2mistral/debian/control

```diff
@@ -11,6 +11,6 @@ Homepage: http://stackstorm.com/
 Package: st2mistral
 Architecture: any
 Pre-Depends: dpkg (>= 1.16.16), python2.7, mistral | python-mistral, ${misc:Pre-Depends}
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}, mistral | python-mistral
```

Comment:
why python-mistral is required and where do we get it from? we don't build python-mistral, it is likely upstream from https://github.com/openstack/python-mistralclient cc @m4dcoder